Calculate/Program.cs

```diff
@@ -0,0 +1,54 @@
+
+using System.Globalization;
+
+namespace Calculate;
+public class Program
+{
+    public Func<string?> ReadLine { get; init; }
```

Comment:
I think you just need to move lines 12 and 13 to 7 and 8 and then you have an empty default constructor
```suggestion
    public Func<string?> ReadLine { get; init; } = Console.ReadLine;
    public Action<string> WriteLine { get; init; } = Console.WriteLine;
    
    public Program() {}
```